syncer/syncer_test.go

```diff
@@ -189,10 +199,14 @@ func TestSynchronize_OnlyOneSynchronize(t *testing.T) {
 func advanceState(t testing.TB, ts *testSyncer, from, to types.LayerID) {
 	t.Helper()
 	for lid := from; lid <= to; lid++ {
-		require.NoError(t, certificates.Add(ts.cdb, lid, &types.Certificate{BlockID: types.EmptyBlockID}))
+		require.NoError(
+			t,
+			certificates.Add(ts.cdb, lid, &types.Certificate{BlockID: types.EmptyBlockID}),
+		)
 		ts.mLyrPatrol.EXPECT().IsHareInCharge(lid)
 		ts.mDataFetcher.EXPECT().PollLayerOpinions(gomock.Any(), lid, false, gomock.Any())
 		ts.mTortoise.EXPECT().TallyVotes(gomock.Any(), lid)
+		ts.mTortoise.EXPECT().OnApplied(gomock.Any(), gomock.Any())
```

Comment:
Would it make sense to specify which parameters were used for the call?

```suggestion
		ts.mTortoise.EXPECT().OnApplied(lid, gomock.Any())
```